upd/cli.py

```diff
@@ -132,18 +133,25 @@ def cli(context, vcf, proband, mother, father, af_tag, vep, min_af, min_gq, logl
     type=click.Path(exists=False),
     default='-',
 )
+
+@click.option('--iso_het_pct',
```

Comment:
```suggestion
@click.option('--iso-het-pct',
```
this will be mapped to `iso_het_pct` in function def so no need to change there